artifacts/contracts/solidity/blind-auction/BlindAuction.sol/BlindAuction.json

```diff
@@ -0,0 +1,298 @@
+{
```

Comment:
Hmm have you rebased the main branch? Before the 0.7 release, I made sure that the "oz" folder has been moved to "openzeppelin" folder. These metadata files shouldn't be here

Reply:
Just did. Thank you.